clients/python-wrapper/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+## v0.11.0
+
+:new: What's new:
+
+- Auth: Authenticate or fallback to AWS IAM authentication (if no other method provided) (#9039)
```

Comment:
we fixed https://github.com/treeverse/lakeFS/issues/7737

Reply:
also https://github.com/treeverse/lakeFS/issues/9063

Reply:
@nopcoder can you write the bug fix/feature change as a comment/suggestion?

Reply:
added